wsinfer/cli/convert_csv_to_geojson.py

```diff
@@ -27,10 +28,14 @@ def _row_to_geojson(row: pd.Series, prob_cols: list[str]) -> dict:
     minx, miny, width, height = row["minx"], row["miny"], row["width"], row["height"]
     coords = _box_to_polygon(minx=minx, miny=miny, width=width, height=height)
     prob_dict = row[prob_cols].to_dict()
-    measurements = [{"name": k, "value": v} for k, v in prob_dict.items()]
+    
+    measurements = {}
+    for k,v in prob_dict.items():
+        measurements[f"{k}"] = v
```

Comment:
this can be simplified.

```suggestion
        measurements[k] = v
```